### ISSUE TITLE
Support Decred

### ISSUE THREAD

**Issue by vctt94** [trezor/connect#562]
decred is supported by trezor-firmware, but it is not supported for trezor-connect right now.

We used trezor.js for our connection, but with that being deprecated, now we need to support decred at trezor-connect, so we can still supporting trezor.

**Comment by szymonlesisz** [comment]
Decred cannot be fully supported since its not compatible with bitcoinjs, or to be precise [trezor fork](https://github.com/trezor/trezor-utxo-lib/) of bitcoinjs lib.
`signTransaction` method is affected since it does validate output addresses and signed transaction using mentioned `@trezor-utxo-lib`

**Comment by vctt94** [comment]
Thanks for letting me know. So should I move this issue there?

**Comment by szymonlesisz** [comment]
yes, `trezor-utxo-lib` is the place to start, but since i'm also the maintainer of it i can tell you that right now i don't have a time to implement this. sorry :/

However any contribution is welcome i can put you thru the process, what needs to be done or changed
